CMakeLists.txt

```diff
@@ -84,11 +84,16 @@ if(WIN32)
         SET(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/Install" CACHE PATH "Cmake install path" FORCE)
     endif()
 
-    # might work with "SHARED" as well.
-    set(PA_LIBRARY_TYPE "STATIC")
+    # PortAudio: use static library
+    set(PA_BUILD_STATIC ON CACHE BOOL "PortAudio static library")
+    set(PA_BUILD_SHARED OFF CACHE BOOL "PortAudio shared library")
+    # Call static library "portaudio_static" since this will also cause the inclusion of the required architecture suffix
+    set(PA_LIBNAME_ADD_SUFFIX ON CACHE BOOL "PortAudio static library suffix")
+    # disable PortAudio install target
+    set(PA_DISABLE_INSTALL ON CACHE BOOL "Disable PortAudio install")
```

Comment:
i think these should go inside the `if(NOT MSYS)` guard. i thought you could get away with not caching these, but apparently not until cmake 3.13: https://cmake.org/cmake/help/latest/policy/CMP0077.html#policy:CMP0077

Reply:
`set(PA_LIBRARY_TYPE "STATIC")` was previously before `if(NOT MSYS)`, that's why I left them there. Wouldn't want these options to be set for all Win32 builds?

Reply:
maybe i'm missing something... what effect do these settings have in MSYS builds?

Reply:
When I moved things to the `external_libraries`'s CMakeLists, I removed MSYS checks. I don't see how they are needed... (?)